packages/icon-button/src/mwc-icon-button-base.ts

```diff
@@ -14,8 +14,11 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-import {ripple} from '@material/mwc-ripple/ripple-directive.js';
-import {html, LitElement, property} from 'lit-element';
+import '@material/mwc-ripple/mwc-ripple.js';
```

Comment:
Google style guide does not like file extensions. Please remove

Reply:
OK, that's really inconsistent 😄 

Are these style guides public btw?